File: 0001 Code Chef/0019 Water Consumption/implementation/0002 c++/water_consumption.cpp
#include <iostream>

int main(){
    int test;
    std::cin >> test;
    while(test > 0){
        int water;
        std::cin >> water;
        if(water >= 2000){
            std::cout << "YES" << std::endl;
        }else{
            std::cout << "NO" << std::endl;
        }
        test = test - 1;
    }
    return 0;
}

File: 0001 Code Chef/0019 Water Consumption/implementation/0003 c#/WaterConsumption.cs
using System;

class WaterConsumption{
    static void Main(string[] args){
        string stringInput;
        stringInput = Console.ReadLine();
        int test = Convert.ToInt32(stringInput);
        while(test > 0){
            stringInput = Console.ReadLine();
            int water = Convert.ToInt32(stringInput);
            if(water >= 2000){
                Console.WriteLine("YES");
            }else{
                Console.WriteLine("NO");
            }
            test = test - 1;
        }
    }
}